src/edu/duke/cs/osprey/astar/conf/scoring/MPLPPairwiseHScorer.java

```diff
@@ -44,7 +44,7 @@
 	private EnergyMatrix emat;
 	private int maxNumIterations;
 	private double epsilon;
-	
+
```

Comment:
whitespace changes are just noise in the commit history. Any way to remove them?